src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -351,6 +351,141 @@ TIP: If you plan to use another DB version, it is important to get the correspon
 
 Moreover, it is also important to notice that the Scheduler and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same scheduler database since they use that one.
 
+=== Data Spaces configuration
+
+link:../user/ProActiveUserGuide.html#_data_spaces[ProActive Data Spaces] is an integrated file-transfer mechanism used by ProActive workflows.
+
+Data Spaces are used to transfer files between the server and <<_glossary_proactive_node,ProActive Nodes>> during workflows tasks execution.
+
+ProActive Data Spaces implementation is based on link:https://commons.apache.org/proper/commons-vfs/[Apache VFS].
+
+The ProActive server defines and starts automatically the following data spaces:
+
+ * A *Global Space* where anyone can read/write files.
+ * An *User Space* which is a personal user data storage.
+
+Global or User Spaces can be configured individually to use one of the following Data Space protocols.
+
+==== ProActive protocol
+
+This is the *default* protocol used by the Global and User Spaces. The ProActive Data Space protocol is able to transfer files either by:
+
+ * a direct copy operation on the file system
+ * a network copy operation through the currently configured <<_available_network_protocols,ProActive Network Protocol>>
+
+Accordingly, a ProActive Node connected to the ProActive server and sharing the same file system (e.g. NFS) will be able to transfer files directly through the file system.
+A ProActive Node connected to the server from a different environment (virtual machine, cloud instance, docker container, etc) will also be able to transfer files to/from the server thanks to the ProActive Data Space protocol.
+
+The ProActive Data Space protocol will use the network protocol configured for the ProActive server (see <<_network_properties>>).
+
+In order to configure on the ProActive server where the file space will be located, you can set the following properties in the `PROACTIVE_HOME/config/scheduler/settings.ini` file:
+
+....
+pa.scheduler.dataspace.defaultglobal.localpath=/path/to/global/space
+pa.scheduler.dataspace.defaultuser.localpath=/path/to/user/space/root
+....
+
+The user space will use the path defined in `pa.scheduler.dataspace.defaultuser.localpath` as its root and create sub-folders according to each user login name.
+
+When these properties are not configured, global and user spaces default locations are:
+
+ * Global Space : `PROACTIVE_HOME/data/defaultglobal`
+ * User Space : `PROACTIVE_HOME/data/defaultuser`
+
+==== SFTP protocol
+
+The SFTP protocol can be used instead of ProActive to deploy the User or Global spaces.
+
+ProActive User Space mapped using SFTP has the following benefits:
+
+ * It allows to map each User Space to the *HOME directory* of the user on the SFTP server.
+ * It allows *file impersonation* : a file copied to the User Space will belong to the user and not to the ProActive server program.
```

Comment:
... belong to the user system account and not to the user ProActive server account.